src/Helper/Quote.php

```diff
@@ -109,16 +109,42 @@ public function __construct(
     public function getValidatedCoupons()
     {
         $coupons = [];
+        $returnInvalidCartRuleCoupon = false;
+        $amount = $this->quote->getSubtotal() - $this->quote->getSubtotalWithDiscount();
+        $couponsIncludeTax = $this->taxHelper->couponInclTax();
         foreach ($this->sgBase->getExternalCoupons() as $coupon) {
+            if ($coupon->getCode() == '1'){
```

Comment:
the `1` needs to become a constant

Reply:
also needs to be a strict comparison `===`

Reply:
done